samples/application_development/code_relocation/prj.conf

```diff
@@ -2,3 +2,5 @@ CONFIG_CODE_DATA_RELOCATION=y
 CONFIG_HAVE_CUSTOM_LINKER_SCRIPT=y
 CONFIG_CUSTOM_LINKER_SCRIPT="linker_arm_sram2.ld"
 CONFIG_COVERAGE=n
+
```

Comment:
The commit message says that no test configuration is changing, but the commit changes 490+ configurations, at least some from their state on master (e.g. "hello_world", which doesn't even use timers).

I think this PR could do with some squashing so the noise from decisions that were later reversed is removed.

Reply:
No test configuration is changing from the previous versions of the PR.

The point is that stuff that is known to pass with the new API still uses the new API and hasn't been needlessly reverted to the old one.